qdrant-landing/content/documentation/cloud/aws-backups.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Back up your AWS cluster
+weight: 65
+---
+
+## Back up your AWS cluster
+
+You can back up your Qdrant clusters deployed on AWS though the Qdrant Cloud
+Dashboard at https://cloud.qdrant.io. This section assumes that you've already
+set up your cluster, as described in the following sections:
+
+- [Create a cluster](/documentation/cloud/create-cluster/)
+- Set up [Authentication](/documentation/cloud/authentication/)
+- Configure one or more [Collections](/documentation/concepts/collections/)
+
+## Configure a backup
+
+After you have taken the prerequisite steps, you can configure a backup with the
+[Qdrant Cloud Dashboard](https://cloud.qdrant.io). To do so, take these steps:
+
+1. Sign in to the dashboard
+1. Select Clusters.
+1. Select the cluster that you want to back up.
+   ![Select a cluster](/documentation/cloud/select-cluster.png)
+1. Find and select the **Backups** tab.
+1. Now you can set up a backup schedule.
+   The **Days of Retention** is the number of days before the backup is deleted.
```

Comment:
We can change `number of days before the backup is deleted` to `number of days after a backup snapshot will be deleted` for better clarity.

Reply:
Thank you for clarifying. I've moved this to the main backup.md file, in the "Configure a backup" section (atm, like 53)